examples/basic/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.5.7"
+  required_providers {
+    digitalocean = {
+      source  = "digitalocean/digitalocean"
+      version = ">= 2.34.1"
```

Comment:
In examples it's better to pin provider version, please remove `>=`. It's fine for terraform itself.